### ISSUE TITLE
Does not support latest java only java 8

### ISSUE THREAD

**Issue by ProGamusian** [toolbox4minecraft/amidst#1014]
Simple as that. It's annoying having to downgrade my java version every time I use it please update it

**Comment by MHPlayer12** [comment]
how do i downgrade to java 8 to get it to work?

**Comment by coldReactive** [comment]
I agree. In addition, if you attempt to select a version of Minecraft 1.17 without the Java SE JDK 16 installed, it will say it needs Java 16 as well.
